tests/dogtag/pytest-ansible/installation/roles/Test_Execution/tasks/configure_ldap.yml

```diff
@@ -1,20 +1,20 @@
 
 - name: Setup DS Service
-  shell: setup-ds.pl --silent --file=/tmp/test_dir/ldap.cfg
+  shell: dscreate fromfile /tmp/test_dir/ldap.cfg
   when: topology  == "topology-01" or topology == "topology-02" or topology == "topology-03" or topology == "topology-04" or topology  == "topology-05" or topology == "topology-ecc"
 
```

Comment:
can we check version here. dscreate and setup method can be picked based on versions.

Reply:
For upstream we have started from 10.6 itself. So version check wont be required. We might have to do that for downstream test.